scripts/markets/validate-other-market-data.ts

```diff
@@ -653,40 +653,21 @@ function getProposalsToValidate(newProposals: Record<string, Proposal>): Set<str
       continue;
     }
 
-    if (JSON.stringify(oldProposals[name].params) !== JSON.stringify(newProposal.params)) {
+    const oldParams = removeIdFromParams(oldProposals[name].params);
+    const newParams = removeIdFromParams(newProposal.params);
+    if (JSON.stringify(oldParams) !== JSON.stringify(newParams)) {
```

Comment:
really generally shouldn't do this for comparison, should at least use a stable stringify that sorts the keys before stringifying. Not really relevant for this PR though.